lib/oli/interop/ingest/processor/pages.ex

```diff
@@ -22,6 +23,34 @@ defmodule Oli.Interop.Ingest.Processor.Pages do
     %Oli.Resources.ExplanationStrategy{type: :after_set_num_attempts, set_num_attempts: 2}
   end
 
+  # Read the collab space config in a robust manner, falling back to defaults if
+  # the entire collabSpace key is missing or if any sub keys are missing
+  defp read_collab_space(%{"collabSpace" => config}) do
+
+    # Read the status in a way that falls back to the proper default
+    # status if no status is specified or if an unsupported status is specified
+    default_status_str = CollabSpaceConfig.default_status() |> Atom.to_string()
+    candidate_status = Map.get(config, "status", default_status_str)
+
+    status = case CollabSpaceConfig.status_values()
```

Comment:
Leave other way of doing this if it's clearer, but anyway looks great 🚀 

![image](https://user-images.githubusercontent.com/42446726/211615030-31c559a8-5f21-442a-b752-9694dc6707b5.png)